app/src/main/java/com/gathering/android/auth/signin/SignInFragment.kt

```diff
@@ -93,11 +97,19 @@ class SignInFragment : DialogFragment() {
                     Log.d("WTF", state.errorMessage.toString())
                     showToast(state.errorMessage)
                 }
-                SignInViewState.NavigateToVerification -> {
```

Comment:
Object are only one instance. they are an alternative to java singleton classes. so you don't need to use `is` for that.